### ISSUE TITLE
Embedded Soft Sensing in Soft Ring Actuator for Aiding with theSelf-Organisation of the In-Hand Rotational Manipulation. (arXiv:2202.12525v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16161]
Embedded Soft Sensing in Soft Ring Actuator for Aiding with theSelf-Organisation of the In-Hand Rotational Manipulation. (arXiv:2202.12525v1 [cs.RO])<br>
https://ift.tt/srPLdyw<br>
<p>This paper proposes a soft sensor embedded in a soft ring actuator with five fingers as a soft hand to identify the bifurcation of manipulated objects during the in-hand manipulation process. The manipulation is performed by breaking the symmetry method with an underactuated control system by bifurcating the object to clockwise or counter-clockwise rotations. Two soft sensors are embedded in parallel over a single soft finger, and the difference in the resistance measurements is compared when the finger is displaced or bent in a particular direction, which can identify the bifurcation direction and aid in the break of symmetry approach without the need of external tracking devices. The sensors performance is also characterised by extending and bending the finger without an object interaction. During an experiment that performs a break of symmetry, manipulated objects turn clockwise and counter-clockwise depending on the perturbation and actuation frequency, sensors can track the direction of rotation. The embedded sensors provide a self-sensing capability for implementing a closed-loop control in future work. The soft ring actuator performance presents a self-organisation behaviour with soft fingers rotating an object without a required control for rotating the object. Therefore, the soft fingers are an underactuated system with complex behaviour when interacting with objects that serve in-hand manipulation field.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
